CarUpdater.cs

```diff
@@ -1,18 +1,33 @@
+using DV.HUD;
+using DV.Utils;
 using HarmonyLib;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using System.Collections.Generic;
-using System.Linq;
 
 namespace DvMod.RemoteDispatch
 {
     public static class CarUpdater
     {
+        private static readonly InteriorControlsManager.ControlType[] WATCHED_CONTROL_TYPES = {
```

Comment:
Please use TitleCase for static readonly (whose contents can be modified at runtime), and only use CAPITAL_CASE for true `const`s.